Kernel/Arch/x86/CPU.h

```diff
@@ -34,7 +35,7 @@ template<Integral T>
 void read_possibly_unaligned_data(u8* where, T& data) requires(sizeof(T) == 8 || sizeof(T) == 4 || sizeof(T) == 2)
 {
     if (((FlatPtr)where % alignof(T)) == 0) {
-        data = *(T*)where;
+        memcpy(&data, where, sizeof(T));
```

Comment:
Think I will switch this over. Used memcpy because it does not change anything codegen-wise and it's more readable, but it did require a new header to be included.